config/config.go

```diff
@@ -266,9 +266,9 @@ func NewAgentConfig(agentIni *File, agentYaml *YamlAgentConfig) (*AgentConfig, e
 	}
 
 	// (Re)configure the logging from our configuration
-	if err := NewLoggerLevel(cfg.LogLevel, cfg.LogFile); err != nil {
-		return nil, err
-	}
+	//if err := NewLoggerLevel(cfg.LogLevel, cfg.LogFile); err != nil {
```

Comment:
I'm not sure this is entirely correct, so the configuration is completely ignored?

Reply:
I'll put it back, but IIRC I did that because it's logging to console no matter what, and on windows, when running as a service console == /dev/null